.github/workflows/ci.yaml

```diff
@@ -66,8 +66,9 @@ jobs:
       - name: Install Helm Unittest Plugin
         run: helm plugin install https://github.com/quintush/helm-unittest
 
+      # Runs all tests/*_test.yaml files by default
       - name: Run Helm Unittest
-        run: helm unittest charts/cadc-tap
+        run: helm unittest charts/*/ applications/*/
```

Comment:
@stvoutsin, any reasons you can think of why we shouldn't do this?

Reply:
I scoped it down to just the dirs with `tests/` subdirs, that makes the output  cleaner.

Reply:
Yes agree with this, great stuff thanks